cmd/virt-launcher/virt-launcher.go

```diff
@@ -263,6 +266,13 @@ func main() {
 
 	log.InitializeLogging("virt-launcher")
 
+	// Block until all requested hookSidecars are ready
+	hookManager := hooks.GetManager()
+	err := hookManager.Collect(*hookSidecars, hookSidecarsCollectTimeout)
```

Comment:
we'll need more than 10 seconds here.   I'd just pass the *qemuTimeout variable in here and get rid of the collectTimeout. That should be fine.